src/app/components/Header/index.tsx

```diff
@@ -309,7 +317,7 @@ export function Header() {
                     isSectionOpen(SECTION_TYPE.TRADE) && 'tw-font-bold'
                   }`}
                 >
-                  <span className="tw-mr-3 tw-cursor-pointer">
+                  <span className="tw-mr-1 2xl:tw-mr-3 tw-cursor-pointer">
```

Comment:
It looks ok on smaller screens but on 1200px it's too crowded. Have you discussed it with @t-z-u ? Because we wanted to have less first level items and we just added another one.